src/views/Layout.jsx

```diff
@@ -1,31 +1,54 @@
 import { Outlet, NavLink } from 'react-router-dom';
 
 import './Layout.css';
+import logo from '../../public/img/logo-bluebg.png';
 
-export function Layout({listToken}) {
+export function Layout({ listToken }) {
 	return (
 		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+			<div className="Layout flex flex-col overflow-y-clip h-screen my-auto">
+				<header className="flex justify-center items-center mt-2">
+					<img
+						src={logo}
+						alt="BerryCart Logo"
+						className="h-20 md:h-64 relative z-10"
+					/>
 				</header>
-				<main className="Layout-main">
+				<main className="relative z-10 mx-auto w-[min(72ch,100%)] h-[65vh] overflow-y-auto border rounded-2xl bg-white pb-10">
 					<Outlet />
 				</main>
-				<nav className="Nav">
-					<div className="Nav-container">
-						{/* this removes the Home navlink from the List and AddItem pages when there is a token assigned */}
-						{!listToken && <NavLink to="/" className="Nav-link">
-							Home
-						</NavLink>}
-						<NavLink to="/list" className="Nav-link">
-							List
-						</NavLink>
-						<NavLink to="/add-item" className="Nav-link">
-							Add Item
-						</NavLink>
-					</div>
-				</nav>
+				<div className='absolute z-0 right-0 bg-right bg-[url("../../img/berry-bg.png")] bg-cover inset-0' />
+				{/* remove navbar from Home when there is no list token */}
+				{listToken && (
+					<nav className="z-5 flex fixed py-4 bottom-0 w-full justify-center bg-red-100">
+						<div className="flex gap-20">
+							{/* this removes the Home navlink from the List and AddItem pages when there is a token assigned */}
+							{!listToken && (
+								<NavLink
+									to="/"
+									className="Nav-link text-slate-950 hover:text-slate-700"
+									activeClassName="active"
+								>
+									Home
+								</NavLink>
+							)}
+							<NavLink
+								to="/list"
+								className="Nav-link text-slate-950 hover:text-slate-700"
+								activeClassName="active"
```

Comment:
It appears that `activeClassName` is not a property on `NavLink` anymore. It looks like it's adding the active class anyway when the link is active, so it should be fine if removed. `className` can also be used as a function that returns a string and one the parameters passed to it is `isActive`